### PR TITLE
Fix examples in documentation

### PR DESCRIPTION
**Pull request by jameslamb**
Some small changes to the documentation. I also re-ran `pkgdown` to reflect these changes in the docs site.

### CODE REVIEW

File: r-pkg/uptasticsearch/man/parse_date_time.Rd [hunk 1]
```diff
@@ -24,8 +24,8 @@ Given a data.table with date-time strings,
 \dontrun{
 
 # Get some data
-result <- es_search(es_host = "http://es.custdb.mycompany.com:9200"
-                 , es_index = 'cust_details')
+someDT <- es_search(es_host = "http://es.custdb.mycompany.com:9200"
```
Comment (austin3dickey) [inline r-pkg/uptasticsearch/man/parse_date_time.Rd]: Did this change happen before this PR and we forgot to roxygenize? (just so I get things straight)

File: r-pkg/uptasticsearch/docs/reference/unpack_nested_data.html [hunk 2]
```diff
@@ -123,7 +123,7 @@ <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examp
   {"film":"Rogue One","pmt_amount":10.25},{"film":"Bridesmaids",
   "pmt_amount":8.75},{"film":"Bridesmaids","pmt_amount":6.25, "matinee": 
   true}]}}}]'</span>
-  , <span class='kw'>keep_nested_data_cols</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>)</div><div class='output co'>#&gt; INFO [2017-07-14 15:03:41] Keeping the following nested data columns. Consider using unpack_nested_data for one:
+  , <span class='kw'>keep_nested_data_cols</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>)</div><div class='output co'>#&gt; INFO [2017-07-15 21:57:24] Keeping the following nested data columns. Consider using unpack_nested_data for one:
```
Comment (austin3dickey) [inline r-pkg/uptasticsearch/docs/reference/unpack_nested_data.html]: Oh this is wild: pkgdown automatically runs examples that aren't wrapped in `/dontrun` and displays the output in the reference page. Crazy!

File: r-pkg/uptasticsearch/man/parse_date_time.Rd (hunk 1 above)
Comment (jameslamb) [inline r-pkg/uptasticsearch/man/parse_date_time.Rd]: no I just made this change in this PR. As I was going through the various examples

File: r-pkg/uptasticsearch/docs/news/index.html [hunk 3]
```diff
@@ -91,13 +91,9 @@
     <div id="uptasticsearch-0-1-0" class="section level1">
 <h1 class="hasAnchor">
 <a href="#uptasticsearch-0-1-0" class="anchor"></a>uptasticsearch 0.1.0</h1>
-<div id="first-release" class="section level2">
+<div id="features" class="section level2">
```
Comment (austin3dickey) [inline r-pkg/uptasticsearch/docs/news/index.html]: How embarrassing, I forgot to regen `docs` in the `docs` PR

Comment (jameslamb) [inline r-pkg/uptasticsearch/docs/news/index.html]: :man_shrugging:

File: r-pkg/uptasticsearch/man/parse_date_time.Rd (hunk 1 above)
Comment (mfrasco) [inline r-pkg/uptasticsearch/man/parse_date_time.Rd]: Yup. I totally messed that up. My bad.

**Comment by austin3dickey** [comment]
hey all I just overhauled this PR. Take another look

**Pull request merged**